src/lib/add-slots.js

```diff
@@ -10,19 +9,21 @@ function isValidSlot (slot: any): boolean {
 
 function addSlotToVm (vm: Component, slotName: string, slotValue: Component | string | Array<Component> | Array<string>): void {
   let elem
-  const vueVersion = Number(`${Vue.version.split('.')[0]}.${Vue.version.split('.')[1]}`)
   if (typeof slotValue === 'string') {
     if (!compileToFunctions) {
       throwError('vueTemplateCompiler is undefined, you must pass components explicitly if vue-template-compiler is undefined')
     }
-    if (slotValue.trim()[0] === '<') {
+    const domParser = new window.DOMParser()
```

Comment:
DOMParser isn't supported in phantomJs, so we need an alternative that will work in phantom

Reply:
Thank you for reviewing.
Is there choice not to support PhantomJS ?
I think it is better to use [Puppeteer](https://github.com/karma-runner/karma-chrome-launcher#headless-chromium-with-puppeteer).

Reply:
A lot of users still use phantom. We could throw an error, or a warning that phantom doesn't support markup in slots, and handle it gracefully

Reply:
Thank you for reply.
I will look for other ways.

Reply:
Can you change the error message to—`option.slots does not support strings PhantomJS. Please use Puppeteer, or pass a component`

Reply:
I am sorry.
I did not notice your comment.
I fixed the error message.